contrib/packaging/rpm/Makefile

```diff
@@ -1,18 +1,17 @@
 include ../../../Makefile.defs
 
-BUILDDIR := "$(CURDIR)/cilium-$(VERSION)"
+CWD := $(CURDIR)
+VERSION := $(shell echo $(VERSION)|cut -c 1-5 -)
```

Comment:
We need an indication for builds that are based on git commits. Ideally including the first 6 characters of the git SHA